Source: written by Pierre Lejeune <superheron@gmail.com>
Upstream: yes, issue #72
Reason: XPI name is wrong, resulting in make install failure
--- eid-mw-3.6.0-scm/_src/eidmw/xpi/build.sh	2011-08-06 15:53:51.316331095 +0200
+++ eid-mw-3.6.0-scm/_src/eidmw/xpi/build.sh	2011-08-06 15:44:12.127469332 +0200
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-EXT_NAME=src
+EXT_NAME=belgiumeid
 BUILDS_DIR=builds
 BUILD_DIR=_build
 
